boards/nxp/frdm_mcxw71/frdm_mcxw71_defconfig

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright 2023 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ARM_MPU=y
+CONFIG_TRUSTED_EXECUTION_SECURE=y
+CONFIG_PINCTRL=y
```

Comment:
pinctrl not allowed in board files

Reply:
can you explain what you mean here, pretty much every other NXP board sets pinctrl = y in their defconfig

Reply:
I did a ` rg "CONFIG_PINCTRL=y" -l | wc -l` and the result was 151.

Reply:
Then your boards are wrong, what does pinctrl have to do with a board? It's a driver thing, drivers should be selecting it, it has nothing to do with a board.

Reply:
The grep I did was a lot more than just NXP boards BTW, also which driver do you think is supposed to select the pinctrl symbol

Reply:
@nordicjm , I created this tracking issue for the NXP boards: https://github.com/zephyrproject-rtos/zephyr/issues/78353 , my opinion though is that this rework is outside the scope of this PR, and would like to request you remove the change request for now, and we will fix this in a follow up PR

Reply:
> The grep I did was a lot more than just NXP boards BTW, also which driver do you think is supposed to select the pinctrl symbol, all of them that use the pinctrl APIs I guess?

Yes

> @nordicjm , I created this tracking issue for the NXP boards: https://github.com/zephyrproject-rtos/zephyr/issues/78353 , my opinion though is that this rework is outside the scope of this PR, and would like to request you remove the change request for now, and we will fix this in a follow up PR

Sure, please fix it soon